cli/cmd/generate.go

```diff
@@ -1,9 +1,16 @@
 package cmd
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
+
+	"github.com/kosyfrances/rundeck-zabbix/lib"
+	"github.com/kosyfrances/rundeck-zabbix/lib/zabbix"
 )
 
+var filePath string
```

Comment:
What is this variable used for?

Reply:
It is used in cli/cmd/jobs.go and cli/cmd/resources.go to store the path to the file given by the user on the command line. It is being defined here because resources and jobs are sub commands of `generate`.

Reply:
Thanks for the catch though, having it there caused a bug for default file paths in both commands.